docs/dapp/tools/other-paratimes/cipher/README.mdx

```diff
@@ -81,8 +81,8 @@ the Cipher transactions using the [`oasis paratime show`] command part of the
 
 :::
 
-[`oasis paratime show`]: ../../general/manage-tokens/cli/paratime.md#show
-[Oasis CLI]: ../../general/manage-tokens/cli/README.md
```

Comment:
I thought we had a preference for relative links.

Reply:
For cross-repo links it's better to use the github.com URL so it also works when viewing markdown files on github.com.

Reply:
fixed the relative and cross-repo links.
@matevz I noticed in this regard that the sapphire-testnet link in ROFL https://github.com/oasisprotocol/oasis-sdk/blob/main/docs/rofl/deployment.md?plain=1#L8 needs to be updated too. To keep in mind when moving the ROFL docs.